.idea/dictionaries/Ilja.xml

```diff
@@ -1,7 +0,0 @@
-<component name="ProjectDictionaryState">
```

Comment:
this file should be commited to the repo, please remove whole .idea catalogue and add it to the .gitignore.